notebooks/official/tabular_workflows/prophet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1345 @@
+{
```

Comment:
<p>1 - resources will be created</p><p>all resources will be given</p><p>will also be exported</p><p> </p><p>Present tense please</p><p> </p><p>2 - max number of workers -&gt; maximum number of workers</p><p> </p><p>3 - Unlike in Vertex Forecasting, all resources will be given hardcoded names&nbsp;under this dataset, and the model artifact will also be exported here.</p><p> </p><p>Vertex Forecasting generates resource names based on the timestamp?</p><p>Vertex Forecasting doesn't export the model artifact? Or it exports it elsewhere?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
Good catch, we can actually remove that line about differences from Vertex Forecasting.